### PR TITLE
Get profile image by uid and url

### PR DESCRIPTION
**Pull request by PfeifferH**

### CODE REVIEW

**Comment by PfeifferH** [comment]
Upon review, the thumbnail images are already being stored in the database. This change just searches for the profile image by uid and url to properly match the correct image.

File: imports/ui/pages/profile.jsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ class _Profile extends Component {
       const UID = UUID.v5({
         namespace: '00000000-0000-0000-0000-000000000000',
         name: fileURL})
-      let image = {UID: UID}
+      let image = {UID: UID, url: fileURL}
```
Comment (ryanmartinneutrino) [inline imports/ui/pages/profile.jsx]: But don't the lines below (if existing) then save the existing image to both the profile and thumbnail?

Comment (ryanmartinneutrino) [reply imports/ui/pages/profile.jsx]: Also, isn't fileURL actually the base64 data, so not what we want to search based on, as it's not stored and would never be found?

Comment (ryanmartinneutrino) [inline imports/ui/pages/profile.jsx]: But don't the lines below (if existing) then save the existing image to both the profile and thumbnail?

Also, isn't fileURL actually the base64 data, so not what we want to search based on, as it's not stored and would never be found?

**Pull request merged**